### PR TITLE
Kprawat

### PR DESCRIPTION
**Pull request by kanchan110295**
Kindly review the changes and merge them

### CODE REVIEW

**Review by svnsairam — CHANGES_REQUESTED** [review]
This commit is failing,  check and create a new pull request

File: src/run.py [hunk 1]
```diff
@@ -45,48 +48,48 @@ async def on_message(message):
         return
     # Step 1: Pre-process message
     msg_content = clean_message(str(message.content))
-    if msg_content.find('hello') != -1:
+    if msg_content.lower().find('hello') or msg_content.lower().find('hi') or msg_content.lower().find('hey') != -1:
```
Comment (vishnuchalla) [inline src/run.py:51]: nice checks. good catch

**Review by vishnuchalla — COMMENTED** [review]
Good to fly.

**Review by vishnuchalla — APPROVED** [review]

**Review by vishnuchalla — COMMENTED** [review]
Resolve the conflicts and update the PR

**Pull request merged**